src/core/ext/lb_policy/grpclb/grpclb.c

```diff
@@ -398,8 +432,8 @@ static void rr_connectivity_changed(grpc_exec_ctx *exec_ctx, void *arg,
     if (error == GRPC_ERROR_NONE) {
       /* RR not shutting down. Mimic the RR's policy state */
       grpc_connectivity_state_set(exec_ctx, &glb_policy->state_tracker,
-                                  rr_conn_data->state, error,
-                                  "rr_connectivity_changed");
+                                  rr_conn_data->state, GRPC_ERROR_REF(error),
```

Comment:
The `GRPC_ERROR_REF()` is probably not needed here, since this only happens if error == GRPC_ERROR_NONE.  But it doesn't hurt anything either.